src/fidesops/models/audit_log.py

```diff
@@ -0,0 +1,24 @@
+from enum import Enum as EnumType
+from sqlalchemy import Column, String, Enum as EnumColumn
+
+from fidesops.db.base_class import Base
+
+
+class AuditLogAction(EnumType):
+    """Enum for audit log actions, reflecting what a  user did"""
+
+    approved = "approved"
+    denied = "denied"
```

Comment:
Just curious are these the only 2 options we anticipate for audit log? Also, do we have a separate ticket for logging `approved`?

Reply:
There isn't a separate ticket for approvals yet. Sean mentioned that we'll eventually have audit logs for most/all of the actions in the UI so I added the action now because there is an approval button too.